packages/tools/pactjs-cli/src/contract-generate/generate.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  FileContractDefinition,
+  generateDts,
+  IContractDefinition,
+  StringContractDefinition,
+} from '@kadena/pactjs-generator';
+
+import { retrieveContractFromChain } from '../utils/retrieveContractFromChain';
+
+import { ContractGenerateOptions } from './';
+
+import { Command } from 'commander';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import mkdirp from 'mkdirp';
+import { dirname, join } from 'path';
+import rimraf from 'rimraf';
+
+export const TARGET_PACKAGE: '.kadena/pactjs-generated' =
```

Comment:
Windows issue here due to incompatible slash separators? could wrap in normalize

Reply:
As far as I can tell this is used with `path.join` so it isn't a problem yes, but if used as-is it could blow up.